apps/system/utils/kdbg_date.c

```diff
@@ -153,156 +155,40 @@ static inline int date_showtime(void)
 	return OK;
 }
 
-static inline int date_settime(int argc, char **args)
+static inline int date_settime(const char *tbuf)
 {
+#if defined(CONFIG_LIBC_LOCALTIME) || defined(CONFIG_TIME_EXTENDED)
+	const char format[] = "%b %d %H:%M:%S %Y %a %j";
```

Comment:
Usages says as shown below:
```
printf("               FORMAT: \"MMM DD HH:MM:SS YYYY DWR DYR D\"\n");
printf("                'month', 'day', 'hour':'minute':'second' 'year' 'day_of_week' 'day_of_year' 'daylight_savings_flag'\n");
printf("                Example: \"Apr 21 10:35:22 1991 Sun 175 1\"\n");
```
Where is the DST at format? Doesn't it support? Then we should modify Usages as well.

Reply:
`strptime ` doesn't support DST, So the DST parsing logic is added here https://github.com/Samsung/TizenRT/blob/8fccc6ecf0d41ebabae4bb2229130ab3fef18bbf/apps/system/utils/kdbg_date.c#L182